core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -261,9 +262,9 @@ public MLong(long i) {
     long l;
   }
 
-  public static Map<KeyExtent,Long> estimateSizes(AccumuloConfiguration acuConf, Path dataFile,
-      long fileSize, Collection<KeyExtent> extents, FileSystem ns, Cache<String,Long> fileLenCache,
-      CryptoService cs) throws IOException {
+  public static Map<KeyExtent,Long> estimateSizes(AccumuloConfiguration acuConf,
+      UnassignedTabletFile dataFile, long fileSize, Collection<KeyExtent> extents, FileSystem ns,
+      Cache<String,Long> fileLenCache, CryptoService cs) throws IOException {
```

Comment:
I noticed this as well but left it alone currently as I wasn't sure if it would buy us much to switch since it's just a file length cache but it might be better to do it just for consistency and have something more safe to use than String for changes in the future.